tasks/deploy/config/base/aerodrome-lp-aero-usdc-91day.ts

```diff
@@ -0,0 +1,89 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { Address, keccak256, parseEther, toBytes, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { AERO_USDC_LP_BASE, THREE_MONTHS } from "../../lib/constants";
+import { BASE_AERODROME_LP_COORDINATOR_NAME } from "./aerodrome-lp-coordinator";
+import { BASE_FACTORY_NAME } from "./factory";
+
+// The name of the pool.
+export const BASE_AERODROME_LP_AERO_USDC_91DAY_NAME =
+    "ElementDAO 91 Day Aerodrome Lp AERO-USDC Hyperdrive";
+
+const CONTRIBUTION = parseEther("100"); // 1e21
```

Comment:
This is actually `1e20`. `parseEther("1") == 1e18`, so `parseEther("10") == 1e19`, and `parseEther("100") == 1e20`